.circleci/config.yml

```diff
@@ -5,7 +5,7 @@ version: 2.1
 #bcs_version: &bcs_version v11.3.0
 
 orbs:
-  ci: geos-esm/circleci-tools@1
+  ci: geos-esm/circleci-tools@2
```

Comment:
@mathomp4 ok with this change?

Reply:
https://github.com/GEOS-ESM/GEOS_OceanGridComp/pull/59

Reply:
Oh. Yes. I made that so that the PR would pass CI. 😄